File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.swift
class Solution {
    func removeDuplicates(_ nums: inout [Int]) -> Int {
        if nums.count == 0 || nums.count == 1{
          return nums.count
        }
        
        var l = 0
        var r = 1
        while r<nums.count{
          if nums[l] == nums[r]{
            nums.remove(at: r)
          } else{
            l+=1
            r+=1
          }
        }
        return nums.count
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
      if not nums:
          return 0
      if len(nums) == 1:
          return 1
      l, r = 0, 1
    

      while r<len(nums):
        while nums[r] == nums[l]:
          nums.pop(r)
        else:
          r+= 1
          l+= 1
      
      print(nums)
      

        

      
        